flexible/mobile-backend/src/main/webapp/Dockerfiles

```diff
@@ -0,0 +1,2 @@
+FROM gcr.io/google_appengine/jetty9-compat:githubheadasync
```

Comment:
That's not really maintained.

Reply:
I'll check with the author on this.

Reply:
Still not maintained.  It probably should be gcr.io/google_appengine/jetty9-compat or gcr.io/jetty9 (nothing in 2 years)or gcr.io/jetty

gcr.io/jetty-compat:release-20190619
or
gcr.io/jetty:2019-06-25-204809 (aka :9)

Reply:
githubheadasync is very old

Reply:
I tested with:
* `gcr.io/google-appengine/jetty`
* `gcr.io/jetty`

The sample works with both, but I rather use `gcr.io/google-appengine/jetty` which is specified in the docs of the [Java/Jetty runtime](https://cloud.google.com/appengine/docs/flexible/java/dev-jetty9).

Reply:
@ricalo, should this change from 
`FROM gcr.io/google_appengine/jetty9-compat:githubheadasync`
To
`FROM gcr.io/google_appengine/jetty9`?

Reply:
Almost, it's `gcr.io/google-appengine/jetty` (without the trailing 9).

Reply:
Updated